.github/actions/sccache-gcloud/action.yml

```diff
@@ -0,0 +1,43 @@
+name: Install SCcache w/ GCloud bucket
+description: Configure Scache to use a Gcloud bucket through workload identity
+
+inputs:
+  GWIP:
+    description: Google Workflow Identity provider
+    required: true
+  GSA:
+    description: Google Service Account
+    required: true
+runs:
+  using: composite
+  steps:
+
+    - name: Auth gcloud
+      id: gauth
+      uses: google-github-actions/auth@ef5d53e30bbcd8d0836f4288f5e50ff3e086997d # @v1
```

Comment:
This diverges from the auth version we use for `build-wasm.yml`: `'google-github-actions/auth@35b0e87d162680511bf346c299f71c9c5c379033' # v1.1.1`